modules/tests/src/test/scala/dev/profunktor/redis4cats/Redis4CatsFunSuite.scala

```diff
@@ -46,6 +47,14 @@ abstract class Redis4CatsFunSuite(isCluster: Boolean) extends IOSuite {
   def withRedis[A](f: RedisCommands[IO, String, String] => IO[A]): Future[Unit] =
     withAbstractRedis[A, String, String](f)(stringCodec)
 
+  def withRedisStream[A](f: Streaming[fs2.Stream[IO, *], String, String] => IO[A]): Future[Unit] = {
+    (for {
+      client <- fs2.Stream.resource(RedisClient[IO].from("redis://localhost"))
+      streams <- RedisStream.mkStreamingConnection[IO, String, String](client, stringCodec)
+      results <- fs2.Stream.eval(f(streams))
+    } yield results).compile.drain.as(assert(true)).unsafeToFuture()
```

Comment:
You can `void` instead of `as(assert(true))` here.

Reply:
done